release/staging-resources-cdk/README.md

```diff
@@ -53,6 +53,12 @@ The following CDK commands all require defining context. The context variables a
 * `archivesBucketName` - The name of the S3 bucket you will use to deploy. This bucket must be in the same region as your stack.
 * `dataPrepperOrganization` - The name of the GitHub organization which has the `data-prepper` repository. This allows you to create staging environments for forks. The default value is `opensearch-project`.
 
+(Optional) If you need a new S3 bucket:
+
+```
+cdk deploy ArtifactsBucketStack --context archivesBucketName={s3-bucket-name} --context dataPrepperOrganization={data-prepper-organization-name}
```

Comment:
Hey @dlvenable, can we simplify the steps to deploy? Why do we need to deploy each resource/stack separately?

My goal is to simplify the steps required to deploy our infrastructure to a single command. Something like: `cdk deploy`.

Reply:
Yes, I'll update the documentation.